src/components/SearchForm.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { useGlobalContext } from '../context'
+export default function SearchForm() {
+  const { setSearchTerm } = useGlobalContext()
+  const searchValue = React.useRef('')
+
+  React.useEffect(() => {
+    searchValue.current.focus()
+  }, [])
+
+  function searchCocktail() {
+    setSearchTerm(searchValue.current.value)
+  }
+  function handleSubmit(e) {
+    e.preventDefault()
```

Comment:
why do we need this?

Reply:
here, event is an object with current property, so whenever submit is hit `searchValue.current` will have value.

We use `e.preventDefault()` because the default browser will try to submit the form and refresh the page, so value doesn't change. 
```
const handleSubmit = (e) => {
    console.log('Hello World') //value doesn't get printed
}
```
fix 

```
const handleSubmit = (e) => {
    e.preventDefault();
    console.log('You clicked submit.');
  }
```
So we must use `preventDefault` method.